content/posts/enable-beta-addons.md

```diff
@@ -0,0 +1,146 @@
+title: Empower your documentation with addons
+date: 2024-03-27
```

Comment:
```suggestion
title: Empower your documentation with addons
date: 2024-04-18
description: Read the Docs Addons let documentation authors add features to their documentation such as analytics, version feedback, and search as you type to documentation in a modular way where authors have control.
```

I verified that this description will automatically get picked up in both the meta description and OpenGraph. This is what will show in previews when the link to the blog is shared in Slack/Discord/FB for example. https://github.com/readthedocs/website/issues/271

Reply:
I'm assuming here that we are capitalizing addons when writing "Read the Docs Addons" but keeping it lowercase when it just referring to addons generally.